lib/dpl/provider/atlas.rb

```diff
@@ -16,14 +16,16 @@ class Atlas < Provider
             chmod +x $HOME/bin/gimme
           fi
 
-          export GOPATH="$HOME/gopath:$GOPATH"
-          eval "$(gimme 1.4.2)" &>/dev/null
+          if [ -z $GOPATH ]; then
+            export GOPATH="$HOME/gopath"
+          else
+            export GOPATH="$HOME/gopath:$GOPATH"
+          fi
+          eval "$(gimme 1.6)" &> /dev/null
 
+          unset GIT_HTTP_USER_AGENT
           go get #{ATLAS_UPLOAD_CLI_GO_REMOTE}
-          pushd $HOME/gopath/src/#{ATLAS_UPLOAD_CLI_GO_REMOTE} &>/dev/null
-          make &>/dev/null
-          cp bin/atlas-upload $HOME/bin/atlas-upload
-          popd &>/dev/null
+          cp $HOME/gopath/bin/atlas-upload-cli $HOME/bin/atlas-upload
```

Comment:
It goes build it, if you run `go get -x ...` you'll see it in the output, https://travis-ci.org/travis-infrastructure/container-tests-org/builds/153417345#L202-L203 deploys successfully after `line 28` does the `cp`